### PR TITLE
REFACTOR-#4: Rename qgrid package to modin_spreadsheet

### PR DESCRIPTION
**Pull request by richardlin047**
This updates the python package import name, widget name, npm package, and jupyterlab plugin from `qgrid` to `modin_spreadsheet`. It also updates the Versioneer version from 0.18 to 0.19.

Example new usage:
`import modin_spreadsheet` (importing python package)
`jupyter nbextension install modin_spreadsheet`

This does NOT update the usage of `qgrid` for these situations:
- Project directory name
- Documentation e.g. README, `docs/`
- File names e.g. `js/src/qgrid.widget.js`
- Variable and function names e.g. `_display_as_qgrid()`
- Github config

I thought it would be easier to refactor the usage `qgrid` in the above sections in a different PR.

### CODE REVIEW

File: js/src/qgrid.widget.js [hunk 1]
```diff
@@ -34,10 +34,10 @@ class QgridModel extends widgets.DOMWidgetModel {
     return _.extend(widgets.DOMWidgetModel.prototype.defaults(), {
       _model_name : 'QgridModel',
```
Comment (devin-petersohn) [inline js/src/qgrid.widget.js:35]: Should these get changed? As well as the name of this file?

Comment (richardlin047) [reply js/src/qgrid.widget.js:35]: Sure, I was thinking to separate the changes, but it's fine to combine them.

Should I stick with `modinspreadsheet` for replacement? Ex: `modinspreadsheet.widget.js` feels verbose. Could use `spreadsheet` for javascript, file name replacement

Comment (devin-petersohn) [reply js/src/qgrid.widget.js:35]: `spreadsheet` makes sense to me.

We can also split the changes. Up to you.

Comment (richardlin047) [reply js/src/qgrid.widget.js:35]: In that case, let's split up minimal package changes with documentation/code refactor. Feel free to merge this if it looks good.

**Review by devin-petersohn — APPROVED** [review]

**Pull request merged**